src/main/java/offTop/SessionHandlerService/KafkaListeners/FocusScoreConsumer.java

```diff
@@ -0,0 +1,25 @@
+package offTop.SessionHandlerService.KafkaListeners;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+import offTop.SessionHandlerService.Services.ConsumerService;
+import offTop.SessionHandlerService.Utils.MessageParserService;
+
+@Service
+public class FocusScoreConsumer {
+    private final Logger logger = LoggerFactory.getLogger(FocusScoreConsumer.class);
+    @Autowired
+    ConsumerService consumerService;
+    @Autowired
+    MessageParserService messageParserService;
+    
+    @KafkaListener(topics = "outgoingFocusScore", groupId = "group_Id")
```

Comment:
should we change group_id to something unique?

Reply:
yeah that's a good idea. I'll chnge it to FocusScore